assets/data/effects/wildermyth-drauven-pcs_outpostCelebration.json

```diff
@@ -28,8 +28,8 @@
 	{
 		"role": "hero3",
 		"template": "HERO_BY_SCORE",
-		"scoreFunction": "SNARK",
-		"scoreThreshold": "60",
+		"scoreFunction": "(10*human)+SNARK",
```

Comment:
Note: The `SNARK` stat can go as high as 100, so if you want to guarantee a human character picking a huge number will be what you want to do (the base game uses 200 in a lot of places)